src/NzbDrone.Core/MediaFiles/TorrentInfo/TorrentFileInfoReader.cs

```diff
@@ -9,9 +11,24 @@ public interface ITorrentFileInfoReader
 
     public class TorrentFileInfoReader : ITorrentFileInfoReader
     {
+        private readonly Logger _logger;
+
+        public TorrentFileInfoReader(Logger logger)
+        {
+            _logger = logger;
+        }
+
         public string GetHashFromTorrentFile(byte[] fileContents)
         {
-            return Torrent.Load(fileContents).InfoHash.ToHex();
+            try
+            {
+                return Torrent.Load(fileContents).InfoHash.ToHex();
+            }
+            catch
+            {
+                _logger.Trace("Invalid torrent file contents: {0}", Encoding.ASCII.GetString(fileContents));
```

Comment:
I assume the value here is we'll be able to log something like HTML content when we expected a torrent?

Similar to logging (IIRC) indexer responses, should we limit this to a maximum size instead of spewing out everything?